starknet-providers/tests/models.rs

```diff
@@ -0,0 +1,42 @@
+use serde::Deserialize;
```

Comment:
The `tests` folder is for [integration tests](https://doc.rust-lang.org/book/ch11-03-test-organization.html#integration-tests). You might want to place these tests elsewhere.